packages/visual-editor/src/internal/puck/constant-value-fields/FAQsSection.tsx

```diff
@@ -60,6 +60,16 @@ const FAQStructArrayField = (): ArrayField<FAQStruct[]> => {
       question: questionField,
       answer: answerField,
     },
+    defaultItemProps: {
```

Comment:
Make as const in the internal package and export so it can be imported into pageSections to avoid duplication. Though it does seem backwards. Thanks shadcli.